docker-compose.yml

```diff
@@ -1,39 +1,59 @@
-version: '3.4'
+## Updated to work with the latest version of 
+## Foreman. Implemented fix for missing tables
+## in the database by running rake. 
+
+## To set the default admin user password run the
+## following after running: docker compose up -d
+## and waiting for all containers to become healthy
+##
+## docker exec container-name-here bundle exec bin/rake permissions:reset username=admin password=admin
+
+version: '3.5'
+name: foreman
+
 services:
   db:
+    image: postgres:17-alpine
+    restart: unless-stopped
+    hostname: db.example.com
     environment:
-      - POSTGRES_USER=foreman
```

Comment:
I still like the dedicated user just for Foreman. That's closer to what we have in production